README.md

```diff
@@ -75,6 +75,27 @@ The deployment's status can be followed through
   one, it also needs to be added to `templates/processbot.yaml`.
  - Secrets are managed through Gitlab.
 
+# Local development
+
+This project is a standard Rust project with some notable requirements
+
+## Requiremnets
```

Comment:
I might also add a `## Debugging` section later on once I work out how it's usually done